artifactory/commands/transferfiles/transfer.go

```diff
@@ -152,6 +157,13 @@ func (tdc *TransferFilesCommand) transferRepos(sourceRepos []string, targetRepos
 			tdc.progressbar.NewRepository(repoKey)
 		}
 
+		if tdc.ignoreState {
+			err = resetRepoState(repoKey)
+			if err != nil {
+				return tdc.cleanup(err, sourceRepos)
```

Comment:
The cleanup already gets triggered when an error is returned from this method
```suggestion
				return err
```